### ISSUE TITLE
keras-CenterNet-master/losses.py", line 28, in reg_l1_loss: gather() got an unexpected keyword argument 'batch_dims'

### ISSUE THREAD

**Issue by lyy-fighting** [xuannianz/keras-CenterNet#14]
This error how to deal with ? According to the Tensorflow 3.6 tf.gather() API the param 'batch_dims' change to 'axis'?  Then axis=1 ?
